### PR TITLE
nerd alert!

### PR DESCRIPTION
**Pull request by aaronj1335**
please take a 👀 . still waiting on official word from @beaktor on location.

### CODE REVIEW

**Comment by beaktor** [comment]
Location to Spredfast, 200 W Cesar Chavez, is approved. :+1:

**Comment by aaronj1335** [comment]
🎸 n roll thanks @beaktor 

just to be clear, is [this](https://github.com/austinjavascript/austinjavascript.github.io/blob/b884477fd8c33fa3fa3ff38aad11cd756e66d6be/_includes/give-em-the-business.html#L7) a good location string? or are you saying `s/Spredfast – 200 West Cesar Chavez Street – 3<sup>rd</sup> Floor of the Silicon Labs building/Spredfast, 200 W Cesar Chavez/` is preferable?

**Comment by beaktor** [comment]
What you have highlighted is great. I'd just add (@ Colorado) to clarify because there's 2 Silicon Labs buildings and both on Cesar Chavez.

**Comment by lawnsea** [comment]
oh god now i have to write this talk

File: _posts/2016-09-15-meetup.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <div>
+    Lon Ingram, <strong>dad to the internet</strong>, is a
+    Princip<strike>led</strike>al UI Engineer at <a
+    href="https://www.retailmenot.com">RetailMeNot</a>. He is fascinated with
+    applying ideas from systems research to the challenge of building cool
+    complicated things on the web.
+  </div>
+</div>
+
+We'd like to thank our gracious hosts [Frog][], and if you have a drink or a slice of pizza, please be sure to say hi and thanks to our sponsors, [Becker Wright Consultants][bwc]!
```
Comment (lawnsea) [inline _posts/2016-09-15-meetup.md]: Our "gracious" "hosts" are "Spredfast" this time

Comment (aaronj1335) [inline _posts/2016-09-15-meetup.md]: done

**Pull request closed without merging**